docker-compose.yml

```diff
@@ -40,14 +40,64 @@ services:
       # Joomlatools Pages
       #- ~/Projects/joomlatools-pages:/var/www/sites/[site]/libraries/joomlatools-components/pages/code
 
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.joomlatools-server.rule=HostRegexp(`{name:(.*)}.test`)"
```

Comment:
@johanjanssens should we also not accept localhost and 127.0.0.1 as the host here? Right now you can only access the container through .test domains via traefik right?

Reply:
No. Containers remains accessible over localhost on their specific IP's as configured in docker. This means:

- http://localhost:8080 => joomlatools-server
- http://localhost:8180 => phpmyadmin
- http://localhost:8280 => traefik
- http://localhost:8380 => mailhog

Same applies when you go through 127.0.0.1, only traffic to localhost over port 80 and traffic to .test over port 80 is proxied by traffice. Also see: http://traefik.localhost/dashboard/#/http/routers for the configured routes

Reply:
Added info to PR.